docs/apache-airflow/index.rst

```diff
@@ -15,65 +15,120 @@
     specific language governing permissions and limitations
     under the License.
 
+What is Airflow?
+=========================================
 
+`Apache Airflow <https://github.com/apache/airflow>`_ is an open-source platform for developing, scheduling,
+and monitoring batch-oriented workflows. Airflow's extensible Python framework enables you to build workflows
+connecting with virtually any technology. A web interface helps manage the state of your workflows. Airflow is
+deployable in many ways, varying from a single process on your laptop to a distributed setup to support even
+the biggest workflows.
 
+Airflow was started in October 2014 by Maxime Beauchemin at Airbnb. It was open source from the very first
```

Comment:
I think this deserves a separate chapter and it's far less important. While it is cool to be able to look at history, it should be something of "if you are really interested in understanding where it came from go and check it"

Reply:
Good point. This information is also under https://airflow.apache.org/docs/apache-airflow/stable/project.html so I'll just remove it from here.